R/readWQPdata.R

```diff
@@ -161,6 +161,13 @@ readWQPdata <- function(..., zip=FALSE, querySummary=FALSE){
   if(querySummary){
     queryHEAD <- HEAD(urlCall)
     retquery <- headers(queryHEAD)
+    countNames <- c('total-site-count', 'nwis-site-count', 'total-result-count', 'nwis-result-count')
```

Comment:
:+1: yep! Should I move the `@importFrom http:HEAD` out of readWQPdata and readWQPqw and just have it for this one new function?